src/main/webapp/WEB-INF/tags/shared/commentsPanel.tag

```diff
@@ -19,10 +19,15 @@
         <c:set var="bodyTitle" value="Comments for students" />
     </c:otherwise>
 </c:choose>
-<div class="panel panel-primary">
-    <div class="panel-heading">
+<div class="panel panel-primary" data-toggle="collapse" data-target="#panelBodyCollapse">
+    <div class="panel-heading cursor-pointer"
```

Comment:
Having the `data-toggle` and `data-target` attributes in the `.panel` element causes the panel to collapse even when clicking on the panel body. This likely caused some of the test failures that I encountered. After making the changes, the templates will have to be updated again.